app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -36,7 +36,11 @@ class RegisterController extends Controller
     protected function redirectTo(){
         if(auth()->user()->role == 'admin'){
             return '/admin/dashboard';
-        }else{
+        }elseif(auth()->user()->role == 'seller'){
+            return 'home/seller';
+
+        }
+        else{
             return '/home';
         }
     }
```

Comment:
This could be a good opportunity to apply Strategy Design Pattern. By giving each involved role (model) an implementation to an "Authenticatables" interface, each one could have their own implementation of the logout method without having check for the current role before logging them out and redirecting them.